source/index-management/tests/createSearchIndex.yml

```diff
@@ -0,0 +1,62 @@
+description: "createSearchIndex convenience helper"
+schemaVersion: "1.10"
+createEntities:
+  - client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      observeEvents:
+        - commandStartedEvent
+  - database:
+      id: &database0 database0
+      client: *client0
+      databaseName: *database0
+  - collection:
+      id: &collection0 collection0
+      database: *database0
+      collectionName: *collection0
+
+runOnRequirements:
+  - minServerVersion: "7.0.0"
+    topologies: [ replicaset, load-balanced, sharded ]
+    serverless: forbid
+
+tests:
+  - description: "no name provided for an index definition"
+    operations:
+      - name: createSearchIndex
+        object: *collection0
+        arguments:
+          definition: { definition: &definition { mappings: { dynamic: true } }  }
+        expectError:
+          isError: true
```

Comment:
I think it makes sense in all the new unified test yaml files to add comments around all the operations expecting errors. Just to note the reasoning - these command don't currently exist outside of Atlas, and we will be updating the unified tests in the future once we have an Atlas instance to test against. I'd also link the new drivers ticket for that work.

Reply:
If these commands don't exist outside of Atlas, do we need new runOnRequirement syntax to restrict these tests to an Atlas environment?

I don't see anything in the PR that says these tests should only be run on Atlas (f.e. some language in the [test README](https://github.com/mongodb/specifications/blob/master/source/index-management/tests/README.rst)).

Reply:
Part 2 of this ticket, which will run the tests against Atlas, would most likely address that. We don't run on Atlas for the first set of tests, which is simply to ignore the errors. We broke it into 2 parts to unblock shell faster.

Reply:
See: https://jira.mongodb.org/browse/DRIVERS-2630